core/integration/module_call_test.go

```diff
@@ -1747,6 +1747,10 @@ func (t *Test) Files() []*dagger.File {
     }
 }
 
+func (*Test) Deploy() string {
+    return "here be dragons!"
+}
```

Comment:
I might be missing something, but I can't see where we check that this property isn't included in the output.

Reply:
Yeah, good catch!